CHANGELOG.md

```diff
@@ -41,6 +42,8 @@ IMPROVEMENTS:
 * `vcd_org_vdc` Add import capability and full read support [#218]
 * Upgrade Terraform SDK dependency to 0.12.8 [#320]
 * `resource/vcd_vapp_vm` has new field `computer_name` [#334]
+* `resource/vcd_catalog_media` Add computed properties (is_iso, owner_name, is_published, creation_date, size, status, storage_profile_name) and full read support [#340]
+* `resource/vcd_catalog_media`: MediaItem state ID changed from colon separated list of catalog name and media name to vCD ID [#340]
```

Comment:
The colon ":" is extra - please remove it here and from a few entries above in this section to make it look consistent.

Reply:
done